### PR TITLE
[CNVS Upgrade] Fix PackagesDetailTab header

### PR DESCRIPTION
**Pull request by jfurrow**
This PR: 
* Fixes a rebase error that got in the header in the `PackagesDetailTab`
* Adds spacing to the `selected` badge
* Removes the leftover `Breadcrumb`

Before:

After:

### CODE REVIEW

File: src/styles/views/packages/styles.less [hunk 1]
```diff
@@ -10,4 +10,63 @@
...
+
+  .selected-badge {
+
+    .badge {
+      margin-right: @base-spacing-unit * 1/4;
+    }
+  }
+}
+
+& when (@packages-view-enabled) and (@layout-screen-small-enabled) {
```
Comment (rcorral) [inline src/styles/views/packages/styles.less]: @jfurrow wouldn't it make more sense to do this:
```
& when (@packages-view-enabled) {
  & when (@layout-screen-small-enabled) {
    ...
  }

  & when (another-screen-size-enbaled) { ... }
```

**Comment by rcorral** [comment]
@jfurrow I'm going to merge this, but it doesn't look correct to me like in your screenshot
img

File: src/styles/views/packages/styles.less (hunk 1 above)
Comment (jfurrow) [inline src/styles/views/packages/styles.less]: I agree. I'd actually like to do away with the `& when (@{whatever}-enabled)` everywhere

**Comment by jfurrow** [comment]
@rcorral forgot to include this in my commit: https://github.com/dcos/dcos-ui/pull/1291

**Pull request merged**